examples/all.yaml

```diff
@@ -5,3 +5,8 @@ sns_topic: arn:aws:sns:ap-southeast-2:0123456789:aem-stack-manager-AemStackManag
 stackmanager_prefix: aem-stack-manager
 s3:
   bucket: aem-stack-manager
+
+poll_timeout:
+  send_message_command_execution:
```

Comment:
This property 'send_message_command_execution' makes me think it's the timeout for the actual message sending, but it's actually not.
Shouldn't this property name reflect the Playbook step name? i.e. it's a 'check', and not a 'send'.

Reply:
Renamed it to check_command_execution